htdocs/compta/facture/card.php

```diff
@@ -3919,7 +3919,7 @@ function js_recalculate_revenuestamp(){
 	print '</tr>';
 
 	// Payments already done (from payment on this invoice)
-	$sql = 'SELECT p.datep as dp, p.ref, p.num_paiement, p.rowid, p.fk_bank,';
+	$sql = 'SELECT DISTINCT pf.rowid as id_paiement, p.datep as dp, p.ref, p.num_paiement, p.rowid, p.fk_bank,';
```

Comment:
Can't see why you get duplicate here. There is a link on pf table and a filter on pf.fk_facture so you should get only one record per entry in paiement (p) table.
Can you paste example of data you get in database ?

Reply:
@eldy 
c'était à cause des personnalisations des dictionnaires par entité, il crée des doublons car pas de filtre par entité sur llx_c_paiement,
mais c'est réglé avec la dernière version 7 de multicompany.

Reply:
"Integrity of data" (in database) must always have priority over "Integrity on screen" (what the user see). 
This is why the DISTINCT is refused here. We must never hide the fact that data are corrupted or not consistent into a database, but instead provide tools to fix this in database.